test/execution/server_test.exs

```diff
@@ -83,7 +82,8 @@ defmodule Wanda.Execution.ServerTest do
       )
 
       assert_receive :wandalorian
-      assert ExecutionResult |> Repo.all() |> Enum.empty?()
+
+      refute ExecutionResult |> Repo.all() |> Enum.empty?()
```

Comment:
This assertion looks pretty weak. Wouldn't be more explicit to check that we have a new execution on running state with that execution_id? This way we test that the execution is in running state as well

Reply:
+1

Reply:
Right, changing to
```elixir
assert %ExecutionResult{execution_id: ^execution_id, status: :running} =
               Repo.one!(ExecutionResult)
```

Reply:
here https://github.com/trento-project/wanda/pull/60/commits/9ec12528f08b35acadd9b31c91f865877e111630